app/src/main/java/com/example/ubclaunchpad/launchnote/recyclerView/DataAdapter.java

```diff
@@ -0,0 +1,56 @@
+package com.example.ubclaunchpad.launchnote.recyclerView;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.example.ubclaunchpad.launchnote.R;
+import com.squareup.picasso.Picasso;
+
+import java.util.ArrayList;
+
+public class DataAdapter extends RecyclerView.Adapter<DataAdapter.ViewHolder> {
+    private ArrayList<AndroidVersion> android;
+    private Context context;
+
+
+    public DataAdapter(Context context, ArrayList<AndroidVersion> android) {
+        this.android = android;
+        this.context = context;
+    }
+
+    @Override
+    public DataAdapter.ViewHolder onCreateViewHolder(ViewGroup viewGroup, int i) {
+        View view = LayoutInflater.from(viewGroup.getContext()).inflate(R.layout.row_layout, viewGroup, false);
+        return new ViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(DataAdapter.ViewHolder viewHolder, int i) {
+
+        viewHolder.tv_android.setText(android.get(i).getAndroid_version_name());
+        Picasso.with(context).load(android.get(i).getAndroid_image_url()).resize(240,220).into(viewHolder.img_android);
```

Comment:
Picasso's a good library for image loading but we're using Glide already, so could you change this to using Glide instead? You can look in the GalleryActivity for an example.